contracts/BIFI/strategies/Solarbeam/StrategySolarbeamMetaStable.sol

```diff
@@ -0,0 +1,369 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-4/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin-4/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "../../interfaces/common/IUniswapRouterETH.sol";
+import "../../interfaces/common/IWrappedNative.sol";
+import "../../interfaces/solar/ISolarStableRouter.sol";
+import "../../interfaces/solar/ISolarChef.sol";
+import "../Common/StratFeeManager.sol";
+import "../../utils/GasFeeThrottler.sol";
+
+struct StablePool {
```

Comment:
is there any difference if you put it outside of the contract?

Reply:
I think the major difference being outside the contract is it allows the struct to be imported from other files. In this case it could have been included inside the contract.